sources/Renderer/Direct3D11/D3D11RenderSystem.cpp

```diff
@@ -551,8 +551,25 @@ void D3D11RenderSystem::ClearStateForAllContexts()
 void D3D11RenderSystem::CreateFactory()
 {
     /* Create DXGI factory */
-    HRESULT hr = CreateDXGIFactory(IID_PPV_ARGS(factory_.ReleaseAndGetAddressOf()));
+#if LLGL_D3D11_ENABLE_FEATURELEVEL >= 2
+
+    HRESULT hr = CreateDXGIFactory2(0, IID_PPV_ARGS(&factory2_));
```

Comment:
These should also all fallback to the next lower factory version if they fail. `LLGL_D3D11_ENABLE_FEATURELEVEL` is only a guard to let LLGL compile on systems with older Windows SDKs. It shouldn't require that the runtime version supports anything higher than Direct3D 11.0.